test/system/070-build.bats

```diff
@@ -464,7 +464,7 @@ Labels.$label_name | $label_value
     local -a files=(
         -test1 -test1.txt
          test2  test2.txt
-         subdir/sub1  subdir/sub1.txt
+         -subdir/sub1 -subdir/sub1.txt
```

Comment:
This really troubles me. Sure, it makes the test pass, but I'm having a lot of trouble accepting it. (But I never reviewed [buildah 3486](https://github.com/containers/buildah/pull/3486). I'm struggling to do so now). @nalind could you take a close look at this diff, with full context above and below, and confirm that it is the desired behavior?

Reply:
Exceptions which use wildcards for any path component other than the final one can end up being ignored.
The version of this test before this patch uses an exception pattern of `*/sub1*` to try to get `subdir/sub1.txt` to be included in the image, and checks that the file is in the produced image.
The upstream change made it so that this exception pattern does not cause the file to be included, so this corrects the test's expectations.
Changing the `.dockerignore` pattern to `!subdir/sub1*` should cause the exception to continue to be made, as an alternate option, if that's preferable.

Reply:
Thank you. That's what I've concluded too, but I really appreciate your confirmation because this just seems like a big change. I will (as a followup) add a test such as `!subdir/sub3` to confirm that explicit `subdir` works.

Reply:
@edsantiago Just to conclude we will have separate new test for this edge case right ? Anything which needs to be done on this PR regarding this edge case ?

Reply:
No, nothing on my end. I'm sorry for causing delays.

Reply:
@edsantiago oh no issues at all in-fact thanks for through audit and making sure everything is on track.